backend/endpoints/messages.py

```diff
@@ -271,6 +271,4 @@ async def get_messages(org_id: str, room_id: str):
             detail="Zc Core failed",
         )
 
-    return paginate(response)
-
-add_pagination(router)
+    return response
```

Comment:
you need to return some other information with this data to enable frontend implementation.
e.g
1. current page (also known as page)
2. size of returned data
3. total number of items in db
4. pointer to the next page
5. pointer to the previous page

Reply:
@Shopiley don't forget this

Reply:
Yeah, I have made another [pull request](https://github.com/zurichat/zc_messaging/pull/165#issue-1467048882) to effect these changes.